### ISSUE TITLE
Windows ARM prebuilt binaries

### ISSUE THREAD

**Issue by nicolasnoble** [glfw/glfw#2163]
This is sort of a feature request.

On the main website, there are prebuilt binaries for MacOS and Windows. The MacOS one contains both Intel and ARM binaries, while the Windows one contains only Intel binaries. With Windows ARM machines starting to crop up, and Microsoft starting to push ARM64 Windows 11, it will start becoming relevant to have ARM builds of GLFW too.

**Comment by Rzself** [comment]
Yes, I also encountered this problem recently, do you have any good solutions

**Comment by gracicot** [comment]
@Rzself you can simply create a build yourself if binary are not provided. It's less convenient but does the trick.

You can also simply install glfw using vcpkg, which will compile it for you.

**Comment by wang-bin** [comment]
this contains glfw static lib for windows arm64 https://github.com/wang-bin/devpkgs/actions/runs/6063275522

**Comment by KimWonjong123** [comment]
I tried both CMake and vcpkg, but none of them worked for me. It is possible to build using either CMake or vcpkg, but if I try to make and build new project using glfw3, it gives me some linker errors.

**Comment by elmindreda** [comment]
I'd be happy to add Windows ARM binaries for future releases but I'm not a Windows ARM developer or user, so I'm not sure what exact variants (compiler version, standard library version, architecture, architecture-specific flags) that would make the most sense to include in the package.

**Comment by JoeTJ3000** [comment]
For what it's worth, I downloaded the current source, ran cmake, and built the solution on Visual Studio 2022 ARM64 without any issues.

I did a quick test with the compiled library and was able to create a window and draw a triangle.

**Comment by mattleibow** [comment]
I suppose my PR does not actually put it on the website... :) But at least I added it in the yaml.
